serviceWorker.js

```diff
@@ -0,0 +1,214 @@
+//Cache polyfil to support cacheAPI in all browsers
+importScripts('./cache-polyfill.js');
+
+var cacheName = 'cache-v4';
+
+//Files to save in cache
+var files = [
```

Comment:
Explicitly declaring these files is no real option in my opinion. Is there another way?

Reply:
I have always used it like this, I will add these files soon btw. And also I will ask around if there is any other way.

Reply:
Ok, no problem. Just to get this started, we could actually stick to that.

Later, it should definitely do this automatically.